.flake8

```diff
@@ -15,7 +15,7 @@ exclude =
     .git,
     .mypy_cache,
     .tox,
-    .env,
+    .env**,
```

Comment:
don't you need this at the front instead? and let's mirror for venv

Reply:
No it's because I made separate envs for the separate tutorials (as in the README); will mirror for venv